datafusion/core/tests/user_defined/user_defined_scalar_functions.rs

```diff
@@ -483,6 +485,196 @@ async fn test_user_defined_functions_with_alias() -> Result<()> {
     Ok(())
 }
 
+/// Volatile UDF that should be append a different value to each row
+struct AddIndexToStringScalarUDF {
```

Comment:
The volatility is important, let's reflect it in the function name -- it's this function's main purpose, not an attribute it happens to have.


`AddIndexToStringVolatileScalarUDF`